app/src/main/java/com/example/drawingactivity/CustomColorViewModel.kt

```diff
@@ -36,11 +38,28 @@ class CustomColorViewModel : ViewModel() {
         currentBitmap = b
     }
 
+    var isDialogOpen = false
+    var currentUserEnteredLayerName: String? = null
+    var layerNumber: Int = 1
+
+    fun addLayer(s: String, b: Boolean) {
```

Comment:
What are these arguments?

Reply:
removed the boolean, renamed string to LayerName